src/main/kotlin/com/frcteam3636/frc2025/subsystems/manipulator/Manipulator.kt

```diff
@@ -36,11 +37,7 @@ object Manipulator : Subsystem {
         LoggedMechanismLigament2d("Manipulator Motor Angle", 40.0, 0.0, 5.0, Color8Bit(Color.kRed))
 
     private fun waitForIntake(): Command = Commands.sequence(
-        Commands.waitUntil { inputs.current > 0.8.amps },
-        Commands.defer({
-            val targetRotations = inputs.position + 1.175.rotations
-            Commands.waitUntil { inputs.position > targetRotations }
-        }, emptySet()),
+        Commands.waitUntil { inputs.laserCanDistance < Inches.of(3.0) },
```

Comment:
This is a bug. It'll trigger as soon as we start intaking. We want to wait for the piece to enter before this.